astropy/table/tests/test_table.py

```diff
@@ -2192,6 +2192,100 @@ def test_kwargs_exception(self):
         assert '__init__() got unexpected keyword argument' in str(err.value)
 
 
+class TestUpdate():
+
+    def _setup(self):
+        self.a = Column((1, 2, 3), name='a')
```

Comment:
Awesome, great idea to just make these simple definitions which are exactly what you need.